README.md

```diff
@@ -1,59 +1,77 @@
+# Games Tracker Project
 ## Description
+> This Project is responsible for helping users to keep up with the latest game releases as well as understand the latest gaming trends.
 
-> This Project is responsible for helping users to keep up with the latest releases as well as understand latest gaming trends.
-## Stakeholder Requirements
+### Stakeholder Requirements
 -  `Gamers` - know what the latests game releases are.
 - `Developers`- understand the market more effectively.
```

Comment:
I think we may have to include a 3rd stakeholder to reflect our presentation, though this is something we will discuss as a group separately